client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -71,7 +71,13 @@ function getHoldMessages( siteSlug, translate ) {
 			description: translate(
 				'Please check your email to confirm your address.'
 			),
-		}
+		},
+		EXCESSIVE_DISK_SPACE: {
+			title: translate( 'Excessive diskspace' ),
+			description: translate(
+				'This site cannot be transferred due to diskspace limitations.'
```

Comment:
There's nothing really reasonable we can ask them to do here. Hopefully we'll be able to remove this in the future.